FakeXrmEasy.Shared/XrmFakedPluginExecutionContext.cs

```diff
@@ -30,7 +30,7 @@ public class XrmFakedPluginExecutionContext : IPluginExecutionContext
         public bool IsExecutingOffline { get; set; }
 
         [DataMember(Order = 7)]
-        public bool IsInTransaction { get; set; }
+        public bool IsInTransaction => Stage == 20 || Stage == 40;
```

Comment:
Hello @iggsn,
I believe that IsInTransaction should check that `Mode==ProcessingStepMode.Synchronous` when stage is equal to 40 so a more correct implementation would be 
```
public bool IsInTransaction 
{
   get
   { 
      return Stage == (int)ProcessingStepStage.Preoperation || Stage == (int)ProcessingStepStage.Postoperation && Mode == (int)ProcessingStepMode.Synchronous;
   }
   set {}
}
``` 
I left the property read/write but ignored the write value to not break the serialization.
Best regards,
Betim.